src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -30,10 +31,10 @@
   <ItemGroup Condition="'$(TargetGroup)' == 'netstandard' OR '$(TargetGroup)' == 'netcoreapp' OR '$(IsUAPAssembly)' == 'true' ">
     <Compile Include="Microsoft.Data.SqlClient.TypeForwards.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true' AND '$(OSGroup)' != 'AnyOS' AND '$(TargetGroup)' == 'netstandard'">
+  <ItemGroup Condition="'$(OSGroup)' != 'AnyOS' AND '$(TargetGroup)' == 'netstandard'">
```

Comment:
Why are we removing IsPartialFacadeAssembly everywhere? Is it not used? I assumed it was related to generating the no-op/PlatformNotSupported assemblies using GenAPI but I never dug into the details.

Reply:
`IsPartialFacadeAssembly` was set when "TargetNetFx" was true, which is never true in our case in any of the DLLs we build. There was also a block for "TargetNetFx=true" that added 2 more dependencies, which is also removed.